topics/data-structure/index.md

```diff
@@ -0,0 +1,8 @@
+---
+aliases: data-structures
+display_name: Data structure
+short_description: Data structure is way of organizing and storing data.
+topic: data-structure
+wikipedia_url: https://en.wikipedia.org/wiki/Data_structure
+---
+ Data structure is a particular way of organizing and storing data in a computer so that it can be accessed and modified efficiently.It is designed to organize data to suit a specific purpose so that it can be accessed and worked within appropriate ways.Examples: array,linked list, class etc.
```

Comment:
Can you remove the leading space?

It looks like this is copied from the Wikipedia page. Maybe we can rephrase? How about:

> A data structure is a particular way storing and organizing data in a computer for efficient access and modification. Data structures are designed for a specific purpose. Examples include arrays, linked lists, and classes.